css/modules.js

```diff
@@ -1,77 +1,63 @@
-const ExtractTextPlugin = require('extract-text-webpack-plugin');
-const autoprefixer = require('autoprefixer');
-const customProperties = require('postcss-custom-properties');
 const Path = require('path');
-
-// Used for autoprefixing css properties (same as Bootstrap Aplha.2 defaults)
-const SUPPORTED_BROWSERS = [
```

Comment:
Can we add the requirement for browserlist to webpack-config insteadso it doesn't need to be included by modules?

Reply:
`browserlist` doesn't get added as a requirement of the module. We add some browserlist _configuration_ to the `package.json` file for each module (it can't be added to _this_ library, I've checked), and is then a dependency for anything (such as babel) which _uses_ it.
Modules themselves don't need any reference to `browserlist` other than the `browserlist` array in `package.json`.